plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/TestsCommon.kt

```diff
@@ -0,0 +1,31 @@
+package org.jlleitschuh.gradle.ktlint
+
+import org.intellij.lang.annotations.Language
+import java.io.File
+
+fun File.buildFile() = resolve("build.gradle")
+
+@Language("Groovy")
+val pluginsBlockWithMainPluginAndKotlinJvm =
+    """
+        plugins {
+            id 'org.jetbrains.kotlin.jvm'
+            id 'org.jlleitschuh.gradle.ktlint'
+        }
+    """.trimIndent()
+
+fun File.defaultProjectSetup() {
+    //language=Groovy
+    buildFile().writeText(
+        """
+            $pluginsBlockWithMainPluginAndKotlinJvm
+
+            repositories {
+                gradlePluginPortal()
+            }
+
+            import org.jlleitschuh.gradle.ktlint.reporter.ReporterType
```

Comment:
🤔 Didn't know you could have an import statement in the middle of a file in groovy scripts. Can you do the same thing in Kotlin?! Huh.